src/ontology/cl-edit.owl

```diff
@@ -28462,6 +28463,15 @@ AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023171 <http:
 AnnotationAssertion(rdfs:label obo:CL_4023171 "trigeminal motor neuron")
 EquivalentClasses(obo:CL_4023171 ObjectIntersectionOf(obo:CL_0000100 obo:CL_4023169))
 
+# Class: obo:CL_4023181 (hypendymal cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0001-7258-9596>) Annotation(oboInOwl:hasDbXref "PMID:26778959"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:9579598"^^xsd:string) obo:IAO_0000115 obo:CL_4023181 "A neurecto-epithelial cell that is part of the basal layer of the subcommissural organ and specialises in the secretion of proteins into the subarachnoid space. Hypendymal cells have similar characteristics to ependymal cells and expresses SCO-spondin.")
```

Comment:
Re: "Hypendymal cells have similar characteristics to ependymal cells and expresses SCO-spondin."
Consider:
"**A hypendymal cell has** similar characteristics to **an** ependymal **cell** and expresses SCO-spondin."
or
"Hypendymal cells have similar characteristics to ependymal cells and **express** SCO-spondin."

I think the former would be preferred since it takes the singular forms.

Reply:
American English spellings are the convention in CL, so "specialises" should be "specializes".

Reply:
> American English spellings are the convention in CL

Fine I'll change it to the wrong spelling :p